lib/components/DropdownMenu/DropdownMenu.test.tsx

```diff
@@ -93,6 +93,6 @@ describe('label', () => {
 describe('hideLabel', () => {
   it('should add correct CSS class to label', () => {
     render(<DropdownMenu label="Non-visible label" options={[]} hideLabel={true} />);
-    expect(screen.getByText('Non-visible label').classList).toContain('sr-only');
+    expect(screen.getByRole('combobox', { name: 'Non-visible label' })).not.toBeNull();
```

Comment:
This `expect` no longer test the original idea, that the correct CSS class is to be found, `sr-only`.
So changing the test name would be appropriate as it no longer uses the CSS-class method for hiding the label in the actual code.

Reply:
I change the test name.